app/providers/yelp.py

```diff
@@ -5,11 +5,9 @@
 
 idPattern = re.compile("/([^/\?]*)(\?.*)?$")
 
-
 def resolve_with_key(key):
     return yelp3Client.request("/businesses/{0}".format(key))
 
-
```

Comment:
I'd prefer single line spaces, but I guess we should talk about that for the future.